docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - ./scripts/docker/env/production/django.env
 
   geoserver:
-    image: geonode/geoserver
+    image: waybarrios/geoserver:piensa
```

Comment:
@waybarrios I'd rather if you can provide a PR with an alias branch for the geoserver repo as well in order to have an official alias image from the geonode docker hub. What do you think?